iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -146,7 +146,7 @@ TEST_F(PoshRuntime_test, GetMiddlewareApplicationIsSuccessful)
 
     ASSERT_NE(nullptr, applicationPortData);
     EXPECT_EQ(m_runtimeName, applicationPortData->m_runtimeName);
-    EXPECT_EQ(iox::capro::ServiceDescription(), applicationPortData->m_serviceDescription);
+    EXPECT_NE(iox::capro::ServiceDescription(), applicationPortData->m_serviceDescription);
```

Comment:
A not invalid check I presume? The expectation somehow changed, I would think a default constructed `ServiceDescription` is invalid and hence we check that we get something valid? If so, we have the `isValid` method I think.